src/views/forms/AccountManagementForms/DepositForm.tsx

```diff
@@ -235,6 +235,10 @@ export const DepositForm = ({ onDeposit, onError }: DepositFormProps) => {
           throw new Error('Missing request payload');
         }
 
+        if (isCctp && !abacusStateManager.chainTransactions.isNobleClientConnected) {
+          throw new Error('Noble client unable to initialize');
+        }
```

Comment:
what's the top reason it can't initialize? wonder if there can be a more user friendly message like can't connect to Noble client or something